specs/Media/mediasource-schema.json

```diff
@@ -0,0 +1,25 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "id": "https://fiware.github.io/dataModels/Media/mediasource-schema.json",
+  "title": "FIWARE - Media / MediaSource",
+  "description": "Source for media events",
+  "definitions":{
+    "MediaSource": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string"
+        },
+        "creationTime": {
+          "type": "string"
```

Comment:
date-time

Reply:
I'm checking this, could it be better to use GSMA-Commons to allow the possibility of a more detailed information in the mediasource , and sendTagsInEvents and parent, to indicate the specifics of the "mediasource". My only concern is the "id" as the mediasource is not expected to be a Orion Entity by itself.